release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 28 March 2024 - Fix - Catalog - Links in organization overview not working correctly [ID_39204]
```

Comment:
Is there a reason why this RN is added at the top, or can we move this below [ID_39214] so that the RNs for the same date are added in numerical order?

Reply:
No specific reason, was a slight oversight from my end when copying over the RNs.